src/Microsoft.DotNet.Interactive.Tests/Connection/SerializationTests.Event_contract_has_not_been_broken.approved.ValueInfosProduced.json

```diff
@@ -2,36 +2,36 @@
   "event": {
     "valueInfos": [
       {
-        "typeName": "String",
+        "typeName": "System.String",
         "name": "a",
         "formattedValue": {
-          "mimeType": "text/plain",
-          "value": "value a"
+          "mimeType": "text/plain\u002Bsummary",
+          "value": "a"
         },
         "preferredMimeTypes": [
           "text/plain"
         ]
       },
       {
-        "typeName": "String",
+        "typeName": "System.Collections.Generic.List\u003CSystem.String\u003E",
```

Comment:
Similar comment as above, the serializer should allow `<` without having to be escaped.

Reply:
The test had a different configuration than the product. Fascinating.